std/typecons.d

```diff
@@ -4319,6 +4321,37 @@ auto nullable(T)(T t)
     assert(b.empty);
 }
 
+// https://issues.dlang.org/show_bug.cgi?id=24403
+unittest
```

Comment:
`std/typecons.d(4325:1)[warn]: A unittest should be annotated with at least @safe or @system`

Reply:
Fixed.

Reply:
Thanks